conductr_cli/constants.py

```diff
@@ -17,7 +16,6 @@
                                        '{}/plugins'.format(DEFAULT_CLI_SETTINGS_DIR))
 DEFAULT_SANDBOX_IMAGE_DIR = os.path.abspath(os.getenv('CONDUCTR_SANDBOX_IMAGE_DIR',
                                                       '{}/images'.format(DEFAULT_CLI_SETTINGS_DIR)))
-DEFAULT_SANDBOX_INTERFACE = os.getenv('CONDUCTR_SANDBOX_INTERFACE', host.loopback_device_name())
 DEFAULT_SANDBOX_ADDR_RANGE = os.getenv('CONDUCTR_SANDBOX_ADDR_RANGE', '192.168.1.0/24')
```

Comment:
Why do we not use `127.0.0.0/24` as the default address range? By doing so the user would not need to add a network interface aliases if he only starts one ConductR instance. Also, out documentation currently uses `127.0.0.1` as the default IP address. Changing it to `192.168.1.1` increases complexity because it would differ from ConductR v1 which might lead to confusions for the user.